CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.1)
 project(RTNeural VERSION 1.0.0)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
RTNeural is intended to be compatible with C++14, so we shouldn't be setting this in the top-level CMakeList.